tests/build/chains.go

```diff
@@ -250,17 +273,21 @@ var _ = framework.ChainsSuiteDescribe("Tekton Chains E2E tests", func() {
 					"MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAENZxkE/d0fKvJ51dXHQmxXaRMTtVz\n" +
 					"BQWcmJD/7pcMDEmBcmk8O1yUPIiFj5TMZqabjS9CQQN+jKHG+Bfi0BYlHg==\n" +
 					"-----END PUBLIC KEY-----")
+				g.GinkgoWriter.Println("Create an invalid public signing key")
```

Comment:
I think "unexpected" is more accurate than "invalid" in this case. The public key is valid. It's just not going to match what was actually used by Chains.